apps/src/codebridge/utils/dragAndDropUtils.ts

```diff
@@ -198,3 +197,36 @@ export const getHighestPriorityCollision = (
 
   return [rectangleCollisions[0]];
 };
+
+// Extend sortableKeyboardCoordinates to support tab and shift+tab to move items.
+export const sortableKeyboardCoordinatesWithTab: KeyboardCoordinateGetter = (
+  event,
+  args
+) => {
+  if (event.code === KeyboardCode.Tab && event.shiftKey) {
+    event.preventDefault();
+    const newEvent = new KeyboardEvent('keydown', {
+      code: KeyboardCode.Left,
+    });
+    // Shift tab is equivalent to left arrow
+    return sortableKeyboardCoordinates(newEvent, args);
+  } else if (event.code === KeyboardCode.Tab) {
+    // Tab is equivalent to right arrow
+    event.preventDefault();
+    const newEvent = new KeyboardEvent('keydown', {
+      code: KeyboardCode.Right,
+    });
+    return sortableKeyboardCoordinates(newEvent, args);
+  } else {
+    // Otherwise, call the original function
+    return sortableKeyboardCoordinates(event, args);
+  }
+};
+
+export const dragAndDropKeyboardCodes = {
+  // Start dragging on 'm', so 'enter' and 'space' can be used to open/close a file/folder.
+  // TODO: expose a menu to users of our keyboard options, until then this is a hidden feature.
+  start: ['KeyM'],
+  cancel: ['Escape'],
```

Comment:
'Escape' in 'm mode seems to be causing the selected file tab or folder/file(s) to change to light gray color. Non-blocking comment, but can will file a ticket.


https://github.com/user-attachments/assets/091cf32a-b237-47a8-a26d-e021119e4b08


https://github.com/user-attachments/assets/060aeec4-5f67-4124-8f89-745a223a1c6e

Reply:
https://codedotorg.atlassian.net/browse/CT-1204